gegelatilib/include/learn/learningAgent.h

```diff
@@ -146,6 +159,18 @@ namespace Learn {
 		*/
 		void init(uint64_t seed = 0);
 
+		/**
+		 * \brief Adds a logger to the loggers vector.
+		 *
+		 * Adds a logger to the loggers vector, so that it will be called in
+		 * addition of the others at some moments. This enables to have
+		 * several loggers that log different things on different outputs
+		 * simultaneously.
+		 *
+		 * @param[in] logger the logger we want to add to the vector.
+		 */
+		void addLogger(Log::LALogger& logger);
```

Comment:
This is strange to add LALogger using a reference, while storing them using a std::shared_pointer<>.
You can use std::reference_wrapper for storing the references to logger in the LearningAgent.